api/web_hub.go

```diff
@@ -60,7 +60,7 @@ func (h *Hub) Stop() {
 	h.stop <- "all"
 }
 
-func (h *Hub) Start() {
+func (h *Hub) StartAndForget() {
```

Comment:
Yup can do, that was one of the ones I wasn't quite sure about but since `PublishAndForget` was above I went with it.